pkg/proto/pbgo/languagedetection/api.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.28.1
-// 	protoc        v3.6.1
+// 	protoc-gen-go v1.30.0
+// 	protoc        v4.25.2
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 Nitpick:** ‏Can we remove these from the PR? Makes it easier to read & probably pings less reviewers